templates/client.mustache

```diff
@@ -30,8 +30,20 @@
 				{{#gitHash}}<div id="releaseHash">{{gitHash}}</div>{{/gitHash}}
 			</div>
 			<nav id="externalLinks">
-				<a href="https://github.com/Zemanzo/web-marbles" id="github" title="Contribute to the source!"><i class="icon-github-circled"></i></a>
-				<a href="https://www.patreon.com/webmarbles" id="patreon" title="Support us on patreon!"><img src="images/patreon.svg" alt="Patreon"></a>
+				{{#invitelink}}
+				<a href="{{invitelink}}" id="discord">
+					<img src="images/wumpus_white.svg" alt="Wumpus">
+					<span>Join the Discord!</span>
```

Comment:
While I do like the new buttons, I preferred the minimalistic look mainly because it wasn't distracting for the gameplay. In my opinion, the buttons should not be in focus that much and perhaps the `<span>` tags for the new buttons should only pop in when hovered?